Sign up

Frost199 opened the pull request.

AbdussamiG commented:
Okay I’ll do that

Frost199 commented:
I just saw your folder structure well, just remove the folder navbar, i isn't necessary, I noticed it is in a folder called Fron-page, the only change you need to make is the renaming your folders to small letters

AbdussamiG committed:
Delete style.css

AbdussamiG commented:
Done, I hope its okay now?

youngGaf committed:
Added mocha

youngGaf committed:
updated app.test.js

tonymj76 committed:
add ejs engine

tonymj76 committed:
change port

tonymj76 committed:
dep

tonymj76 committed:
add files

tonymj76 committed:
update

tonymj76 committed:
hotfix

tonymj76 committed:
add comment

youngGaf committed:
commits to match upstream

youngGaf committed:
commits to match upstream

youngGaf committed:
Commit to test

tonymj76 committed:
Merge pull request #11 from hngi/develop

add comment

Olanetsoft committed:
Fixed registered engine path

Commit:
assets folder added

Frost199 committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into git-ignore

Commit:
added boilerplate for index page and stylesheet

thebolarin committed:
Signup test

thebolarin committed:
Merge branch 'master' of https://github.com/hngi/Team-Incredible-BE into dev

Commit:
linked stylesheet to page

AbdussamiG committed:
added for the navbar

Dami-js committed:
Merge pull request #1 from AbdussamiG/patch-1

added for the navbar

thebolarin committed:
Delete this file

AbdussamiG committed:
added the html code for the navbar

AbdussamiG committed:
requested update

robertechie committed:
Merge pull request #16 from thebolarin/dev

Signup test ,fix, feat

AbdussamiG committed:
Removed meta tags

tonymj76 committed:
Merge pull request #14 from Olanetsoft/new-branch

Fixed registered engine path

robertechie committed:
Merge pull request #9 from youngGaf/login-appTest

Login app test

tonymj76 commented on Pages/front-page/css-style/styles.css:
```diff
@@ -0,0 +1,68 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap');
```
please pull from the recent branch and modify your changes

tonymj76 commented on Pages/front-page/index.html:
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```
we are using ejs not vanilla html

Dami-js committed:
Merge pull request #2 from AbdussamiG/patch-3

added the html code for the navbar

Commit:
created public folder for assets and moved index stylesheet

robertechie committed:
Merge pull request #15 from Frost199/git-ignore

chore: Update to .gitignore

Commit:
Replaced body-parser middleware with the express.json()

Commit:
Variable typo fixed

Commit:
abbreviation used in route.js changed to full word

Commit:
excessive spacing removed

Commit:
dead dependency removed

Commit:
nodemon moved to devDependencies

Frost199 committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into git-ignore

Commit:
added images folder

Olanetsoft committed:
Updated Readme with Postman Login result

tonymj76 committed:
Merge pull request #18 from sixpathdev/develop

Refactored codes and dependencies

tonymj76 committed:
Merge pull request #19 from Olanetsoft/new-branch

Updated Readme with Postman Login result

Olanetsoft committed:
Updated Readme with Postman SignUp result

tonymj76 committed:
Merge pull request #20 from Olanetsoft/new-branch

Updated Readme with Postman SignUp result

ogunsoladebayo committed:
added about page rendering, setup about page and ejs partials

Frost199 committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into git-ignore

ogunsoladebayo committed:
added about page rendering, setup about page and ejs partials

robertechie committed:
Merge pull request #21 from ogunsoladebayo/develop

seen

tonymj76 committed:
add urlencoded

Olanetsoft committed:
Updated Pages Folder with Login and register page

tonymj76 committed:
add urlendcoded

tonymj76 committed:
hot fix

Olanetsoft committed:
Updated Readme with Postman Login result- Conflict Fixed

Frost199 committed:
Merge branch 'homepage' of https://github.com/Dami-js/Team-Incredible-BE into home-page

Hybee22 committed:
test: adding test for signup functionality

Dami-js committed:
Merge pull request #3 from Frost199/home-page

chore: Add container class for styling

Commit:
removed duplicated styles

ogunsoladebayo committed:
updating about page

tonymj76 committed:
Merge pull request #22 from Olanetsoft/new-branch

Updated Pages Folder with Login and register page

youngGaf committed:
Commit on unit login test by Gafar_01

timothymayor commented:
Please, kindly you push to the develop branch not master!

timothymayor committed:
Merge pull request #23 from Hybee22/hybee-test

test: adding test for signup functionality

AbdussamiG commented:
Alright I’ll do that

timothymayor committed:
Merge pull request #24 from youngGaf/master

Commit on unit login test using mocha by Gafar_01

AbdussamiG committed:
the index.html code for the navbar

AbdussamiG committed:
the style.css code for the navbar

ogunsoladebayo committed:
Merge remote-tracking branch 'upstream/develop' into develop

ogunsoladebayo committed:
updating about page

Dami-js committed:
Merge pull request #6 from AbdussamiG/master

The complete navbar code

Commit:
removed uneccessary pages folder

Frost199 committed:
fix: Proper rendering of navbar

A slight fix for the class attribute of the ul tag.
Initial attribute was links but was replaced to class
as links isn't a tag attribute

AbdussamiG committed:
update to the navbar html code

robertechie committed:
Merge pull request #25 from ogunsoladebayo/develop

Good work

Dami-js committed:
Merge pull request #7 from AbdussamiG/patch-5

update to the navbar html code

AbdussamiG committed:
update to the navbar style.css file

Frost199 committed:
Merge branch 'homepage' of https://github.com/Dami-js/Team-Incredible-BE into home-page

Dami-js committed:
Merge pull request #8 from AbdussamiG/patch-6

update to the navbar style.css file

Dami-js committed:
Merge pull request #9 from Frost199/home-page

fix: Proper rendering of navbar

Myspheet committed:
Added validation for the signup page

Commit:
refactored navbar view

thebolarin committed:
More test cases

thebolarin committed:
More test cases

imykie committed:
Adding Dockerfile

imykie committed:
modifiying dockerfile and adding gitignore

programmerjide committed:
Feat: added hero section in index.ejs and index.css with hero-image.png

Dami-js committed:
Merge pull request #12 from programmerjide/homepage

Feat: added hero section in index.ejs and index.css with hero-image.png

tonymj76 committed:
add pages route

tonymj76 committed:
hot fix

tonymj76 committed:
house keeping

tonymj76 committed:
index page

tonymj76 committed:
about

tonymj76 committed:
login page

tonymj76 committed:
register page

tonymj76 committed:
components

t-bello7 committed:
Added footer to index.ejs file

tonymj76 committed:
Merge pull request #27 from thebolarin/dev

Unit test for signup page

Commit:
refoactored hero section

Dami-js committed:
Merge pull request #14 from kyle2f/patch-3

Added footer to index.ejs file bytomi and charles

Myspheet committed:
Merge branch 'develop' into confirmPasswords

imykie committed:
about build docker image on aws ec2 instance

imykie committed:
Merge branch 'develop' into docker

tonymj76 committed:
Merge pull request #26 from Myspheet/confirmPasswords

Added validation for the signup page

Commit:
added the about section and footer section

Commit:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into homepage

Frost199 committed:
cleanup project files

imykie committed:
Adding run_docker shell script

imykie committed:
Merge branch 'develop' into docker

tonymj76 committed:
Merge pull request #29 from MichaelOlatunji/docker

Dockerizing application

Frost199 committed:
build: Setup github workflow

Add a github workflow for test builds

Frost199 committed:
feat: Add github build

Frost199 committed:
Merge branch 'github-workflow' of https://github.com/Frost199/Team-Incredible-BE into home-page

Frost199 committed:
fix: resolve merge conflict with develop

Frost199 committed:
fix: modify github workflow

Making sure builds are being triggered on push and pull to
master and develop branch

Commit:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into homepage

Olanetsoft committed:
.

Olanetsoft committed:
Merge remote-tracking branch 'upstream/develop' into develop

Commit:
moved header, navbar and footer into partials

Frost199 committed:
Merge branch 'homepage' of https://github.com/Dami-js/Team-Incredible-BE into home-page

Frost199 committed:
fix: updated branch

This branch has been updated with the changes made with the homepage
team and was requrired to be updated by @tonymj76

tonymj76 committed:
Merge pull request #37 from Dami-js/homepage

Homepage

tobecci committed:
SETUP: installed dependencies with npm

Frost199 committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into git-workflow

Frost199 committed:
fix: Update to the css

Breakpoints were not rendering well so I made a slight change to make
the responsive nataure of the page look good, meta tags were not
supporting IE and Edge and also not having proper minimun and maximum
scale

tonymj76 committed:
Merge pull request #33 from Frost199/home-page

chore: Add github workflow

Olanetsoft committed:
Merge remote-tracking branch 'upstream/develop' into develop

Frost199 committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into git-workflow

Frost199 committed:
fix: Update package.json

Package.json file updates to run all test found in th unit test folder

Frost199 committed:
test: fix server shutdown

The server shutdown was implemented so as to close every ports after
each test has been run, this helps to prevent the tests from running in
infinite loops

Josh4324 committed:
feat/header

fosajeff committed:
Merge pull request #1 from Josh4324/joshdev

feat/header

Frost199 committed:
docs: Update README

The README file has been updated to explain the project in full
it shows our build status, and also what the project needs for it to be
properly set up

Kindly update this readme as development goes on

Commit:
implemented unit test recommendationd

Commit:
putting things together

fosajeff committed:
add managing users micro service section

Josh4324 committed:
bug-fix

mosiac05 committed:
mosiac05: Added view more button and get amplified area

fosajeff committed:
Merge pull request #2 from Josh4324/joshdev

bug-fix

fosajeff committed:
Merge pull request #3 from mosiac05/feat/about-page-view

mosiac05: Added view more button and get amplified area

tonymj76 committed:
Merge pull request #44 from Frost199/readme

updates to the readme

tobecci committed:
FEAT: Creating the login page

youngGaf committed:
commit to unit login test

dreywandowski committed:
Dockerized microservices for managing company settings

dreywandowski committed:
Dockerized microservices for managing company settings

tobecci committed:
FIX: |Login page| uncommented image

fosajeff committed:
add media queries for about page

Commit:
adjusted navbar on landscape mobile view

dreywandowski committed:
Third microservice

dreywandowski committed:
Merge branch 'feat/about-page-view' into feat/about-page-view

youngGaf committed:
Commint on unit test login

youngGaf committed:
commit to unit login test

robertechie committed:
Merge pull request #51 from youngGaf/loginTest

Login test

fosajeff committed:
Merge pull request #5 from dreysongz/feat/about-page-view

Third microservice

Commit:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into homepage

Olanetsoft committed:
Merge remote-tracking branch 'upstream/develop' into develop

Frost199 committed:
Merge pull request #52 from Dami-js/homepage

Homepage

Sahmie committed:
add microservice section for authentication with images

Olanetsoft committed:
Merge remote-tracking branch 'upstream/develop' into develop

Sahmie committed:
fix conlict in aboutme.ejs

Sahmie committed:
remove authorization section images

Hybee22 committed:
Merge pull request #1 from hngi/develop

updating forked repository branch with the main repository develop branch

fosajeff committed:
Merge pull request #6 from Sahmie/feat/about-page-view

Feat/about page view

Hybee22 committed:
test: added tests for the signup ui components and api POST method

onshimiye committed:
feat: added missing images

onshimiye committed:
feat: spacing between image and text

robertechie committed:
Merge pull request #53 from Hybee22/hybee-test-scripts

test: added scripts to test the signup UI components and API POST

fosajeff committed:
Merge pull request #7 from onshimiye/feat/about-page-view

Feat/about page view

fosajeff committed:
refactor code

Hybee22 committed:
test: updated the test script and package.json file

fosajeff committed:
fix conflict

Olanetsoft committed:
Merge remote-tracking branch 'upstream/develop' into develop

robertechie committed:
Merge pull request #54 from Hybee22/hybee-test-scripts

test: updated the test script and package.json file

Olanetsoft committed:
Merge remote-tracking branch 'upstream/develop' into develop

fosajeff committed:
refactor

fosajeff committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into feat/about-page-view

Oluwatobilobaoke committed:
optimized the images on the frontpage

Oluwatobilobaoke committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into homepage-image-optimization

tonymj76 committed:
Merge pull request #56 from fosajeff/feat/about-page-view

Feat/about page view

Hybee22 committed:
test: updated the test script and package.json file

Hybee22 committed:
Merge pull request #2 from hngi/develop

updating forked repository with main repo updates

tonymj76 committed:
Merge pull request #58 from hngi/develop

version 0 deployment

tobecci committed:
FEAT: Finished building Login page [Desktop]

robertechie committed:
Merge pull request #57 from Dami-js/homepage-image-optimization

Homepage image optimization

robertechie committed:
Merge pull request #59 from Hybee22/hybee-test-scripts

test: updated the test script

tobecci committed:
FIX: removed commented code

Commit:
fixed call to action button mobile display

Xemicolon committed:
Sign Up page

robertechie committed:
Merge pull request #60 from Xemicolon/develop

Sign Up page

Commit:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into homepage

Oluwatobilobaoke committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into homepage

tonymj76 committed:
Merge pull request #61 from Dami-js/homepage

fixed call to action button mobile display

tobecci committed:
SYNC: Merge remote-tracking branch 'upstream/develop' into develop

tonymj76 committed:
update dep

Olanetsoft committed:
Merge remote-tracking branch 'upstream/develop' into develop

timothymayor committed:
Merge pull request #62 from tobecci/develop

Created Login Page

Olanetsoft committed:
Merge remote-tracking branch 'upstream/develop' into develop

tonymj76 committed:
Merge pull request #63 from hngi/master

Master

tonymj76 committed:
Merge pull request #64 from hngi/develop

v2 login n reg

Oluwatobilobaoke committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into homepage

tonymj76 committed:
rm

Olanetsoft committed:
Merge remote-tracking branch 'upstream/develop' into develop

dreywandowski committed:
fixed img responsiveness for 3rd microservice

dreywandowski committed:
Merge branch 'feat/about-page-view' into feat/about-page-view

Oluwatobilobaoke committed:
Optimized The About page

Uploaded the images to cloudinary,
now the page loads more faster than before

Xemicolon committed:
Sign Up page completed

timothymayor committed:
Merge pull request #65 from Xemicolon/develop

Sign Up page completed

timothymayor committed:
Merge pull request #66 from Dami-js/about-page-optimization

About page optimization

Frost199 committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into readme

Xemicolon committed:
Minor fix on sign up page

Hybee22 committed:
Merge pull request #3 from hngi/develop

updating forked repo with latest commits in the main repository

Oluwatobilobaoke committed:
Optimzed Register Page

Uploaded the images to cloudinary for proper optimization

timothymayor committed:
Merge pull request #68 from Dami-js/signup-page-optimization

Optimzed Register Page

timothymayor committed:
Merge pull request #67 from Xemicolon/develop

Minor fix on sign up page

Olanetsoft committed:
Merge remote-tracking branch 'upstream/develop' into develop

Frost199 committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into readme

dreywandowski committed:
fixed mobile overflow

Olanetsoft committed:
IMPLEMENTED THE LOGIN PAGE RESPONSIVENESS AND LAYOUTS ALSO INCLUDED

Olanetsoft committed:
FIXED TYPO

timothymayor committed:
Merge pull request #69 from Olanetsoft/develop

UPDATED LOGIN PAGE RESPONSIVENESS AND LAYOUTS

Frost199 committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into readme

Xemicolon committed:
Minor fix

Xemicolon committed:
Minor fix for spacing

timothymayor committed:
Merge pull request #70 from Xemicolon/develop

Minor fix

Frost199 committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into readme

Xemicolon committed:
Fixed input width on 767 screen

Olanetsoft committed:
FIXED MARGIN ISSUES IN LOGIN PAGE

Frost199 committed:
Merge pull request #71 from Xemicolon/develop

Fixed input width on 767 screen

Frost199 committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into readme

Frost199 committed:
Merge pull request #72 from Olanetsoft/develop

FIXED MARGIN ISSUES IN LOGIN PAGE

Frost199 committed:
fix: Mobile view for login

Form is centered for mobile view

Frost199 committed:
fix: minimal changes to signup

Olanetsoft committed:
INCLUDED POST ABILITY TO LOGIN PAGE

Frost199 committed:
Merge pull request #73 from Frost199/sign-up

Sign up

timothymayor committed:
Merge pull request #74 from Olanetsoft/develop

INCLUDED POST ABILITY TO LOGIN PAGE

fosajeff committed:
Merge pull request #9 from dreysongz/feat/about-page-view

fixed mobile overflow issue

fosajeff committed:
fix merge conflict

Xemicolon committed:
I saw terrible things

Frost199 committed:
Merge pull request #77 from Xemicolon/develop

Minor fix

Xemicolon committed:
Fixed the sign up page

Xemicolon committed:
Fixed all the issues on sign up page

tonymj76 committed:
hotfix

Hybee22 committed:
Merge pull request #4 from hngi/develop

updating forked repo before pushing testSignup.js file

Frost199 committed:
Merge pull request #78 from Xemicolon/develop

Sign up page is fixed

Hybee22 committed:
test: merged test script

Frost199 committed:
Merge pull request #79 from Hybee22/test-signup

test: merged test script for signup

fosajeff committed:
add page responsiveneess for mobile

fosajeff committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into feat/about-page-view

tonymj76 committed:
Merge pull request #80 from fosajeff/feat/about-page-view

Feat/about page view

Frost199 committed:
merging unrelated histories

Frost199 committed:
Merge branch 'develop' of https://github.com/hngi/Team-Incredible-BE into sign-up

Frost199 committed:
fix: merge issues with Register.ejs
